problems/countThrees.js

```diff
@@ -13,8 +13,15 @@
 * returns: 0
 */
 
-function countThrees() {
-
+function countThrees(n, target) {
+    let output = 0;
+    if (n.toString().length) {
```

Comment:
we would want to use the length of this parameter for looping, the if statement could of been used check if the current digit is a 3 or not

Reply:
Thank you Sparky; tried using that method but it wouldn't consider a number like 303  I was only getting one three